.travis.yml

```diff
@@ -66,7 +66,7 @@ jobs:
 script:
   # brew sometimes fails on something after the python installation -- but we don't want the build to fail
   - ./travis-scripts/travis-setup-osx-python.sh || true
-  - sudo python3 -m pip install Cython==0.29 --install-option="--no-cython-compile"
+  - sudo python3 -m pip install Cython==0.29
```

Comment:
Since it turned out this didn't help, could you put that option back in? It's supposed to speed up the install.